### PR TITLE
Move url to option

### PR DESCRIPTION
**Pull request by IrinaShkviro**
Moved Crash2 collection server url to the option and switched from staging to production. It is only available in kokoro builds for now. So in other builds crash dumps are written locally and aren't send to the collection server, but it could be enabled by providing the option `-o crashdump_server =<crash_dump_server_url>`.

### CODE REVIEW

File: OrbitCore/CrashHandler.cpp [hunk 1]
```diff
@@ -4,13 +4,14 @@
...
 
+#include "client/settings.h"
+
 #include "Core.h"
 #include "OrbitBase/Logging.h"
 #include "OrbitDbgHelp.h"
 #include "ScopeTimer.h"
 #include "TcpClient.h"
 #include "Version.h"
-#include "client/settings.h"
```
Comment (beckerhe) [inline OrbitCore/CrashHandler.cpp:13]: I this a clang-format related change?

Comment (beckerhe) [reply OrbitCore/CrashHandler.cpp:13]: If not I would consider removing it.

Comment (IrinaShkviro) [reply OrbitCore/CrashHandler.cpp:13]: I changed it just to have a separate group with libraries' headers that are not from our project (`crashpad` library in this case).

**Comment by IrinaShkviro** [comment]
> This looks really good. But I think you still need `fetch_keystore`-entries in our kokoro config files.

Yess, thank you! Fixed now

**Pull request merged**